dwave/cloud/client.py

```diff
@@ -646,24 +651,62 @@ def solvers(self, name=None, qpu=None, software=None, vfyc=None,
             software (bool, default=None):
                 Should solver be software based?
 
-            vfyc (bool, default=None):
-                Should solver work on "virtual full-yield chip"?
-
-            flux_biases (bool, default=None):
-                Should solver accept flux biases?
-
-            anneal_schedule (bool, default=None):
-                Should solver accept anneal schedule?
-
-            num_qubits (int/[int,int], default=None):
-                What's the range (or exact number) of qubits solver should handle?
-
             online (bool, default=True):
                 Should solver be online?
 
             refresh (bool, default=False):
                 Force refresh cached list of solvers/properties
 
```

Comment:
Here I'd place the **filters parameter with its explanation

Reply:
Resolved now.